### ISSUE TITLE
Page feedback - exporter (delimiters)

### ISSUE THREAD

**Issue by sparecycles** [open-telemetry/opentelemetry-specification#4296]
Suggested improvements for page: https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specifying-headers-via-environment-variables

The "Specifying headers via environment variables" section mentions **semi-colon delimited** values:

> The OTEL_EXPORTER_OTLP_HEADERS, OTEL_EXPORTER_OTLP_TRACES_HEADERS, OTEL_EXPORTER_OTLP_METRICS_HEADERS, OTEL_EXPORTER_OTLP_LOGS_HEADERS environment variables will contain a list of key value pairs, and these are expected to be represented in a format matching to the [W3C Baggage](https://www.w3.org/TR/baggage/#header-content), except that additional **semi-colon delimited** metadata is not supported, i.e.: `key1=value1,key2=value2`. All attribute values MUST be considered strings.

but the i.e., example is comma-delimited.  So is that an example of what to do or what not to do? (that is the question).

**Comment by pellared** [comment]
@sparecycles, please read it another time

> these are expected to be represented in a format matching to the [W3C Baggage](https://www.w3.org/TR/baggage/#header-content), **except that additional semi-colon delimited metadata is not supported**

Therefore, it is an example of what to do.

**Comment by trask** [comment]
I agree it reads a bit confusing, I think a clarification PR would be welcome
